presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/SymbolCardinalityMatcher.java

```diff
@@ -30,9 +30,15 @@
     }
 
     @Override
-    public boolean matches(PlanNode node, Session session, Metadata metadata, ExpressionAliases expressionAliases)
+    public boolean shapeMatches(PlanNode node)
```

Comment:
These changes should go in the commit that introduced this class: 37e25a974df9e559130a372b8065f047adedc13a